packages/vue/src/components/organisms/SfTabs/SfTabs.js

```diff
@@ -1,74 +1,25 @@
-import Vue from "vue";
-import SfTab, { TAB_COMPONENT_NAME } from "./_internal/SfTab.vue";
-import SfTabsNavigation from "./_internal/SfTabsNavigation.vue";
-
-Vue.component("SfTab", SfTab);
-Vue.component("SfTabsNavigation", SfTabsNavigation);
+import SfTab from "./_internal/SfTab.vue";
 
 export default {
   name: "SfTabs",
-
-  provide() {
-    return {
-      changeTab: this.changeTab,
-      sharedState: this.sharedState,
-      fireEvent: this.fireEvent,
-      listenEvent: this.listenEvent
-    };
-  },
-
-  model: {
-    prop: "value",
-    event: "change"
+  components: {
+    SfTab
   },
 
   props: {
-    /**
-     * Value of selected option, this prop has v-model support
-     */
-    value: {
-      type: [Number, String],
-      default: null
+    /** Array of tabs objects  */
+    tabs: {
+      type: Array,
+      default: () => []
```

Comment:
Hey @Michal-Dziedzinski ,
These changes are in disagreement of conventions for loop components: #254
I think you should just edit mobile view without changing the `SfTabs` api and functionality